include/rocksdb/db.h

```diff
@@ -1680,6 +1682,14 @@ class DB {
   }
 #endif  // ROCKSDB_LITE
 
+  // Create a default trace replayer.
+  virtual Status NewDefaultReplayer(
```

Comment:
We should mention that this Replayer can only replay the trace to this DB.

Reply:
Do you mean the replayer can only replay trace file generated from the same db, or the replay will only be executed on this db? It is possible to create the trace file from a different db and replay on another db.

Reply:
> Do you mean the replayer can only replay trace file generated from the same db, or the replay will only be executed on this db? It is possible to create the trace file from a different db and replay on another db.

No I mean, all the traces can only be replayed on a certain DB which create this replayer. Initially, replayer is independent to the DB. But now, the replayer created from this interface is fixed to the DB.